stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -155,20 +198,46 @@ public void createToken(final Card card, final String publishableKey, final Exec
     }
 
     private Card androidCardFromStripeCard(com.stripe.model.Card stripeCard) {
-        return new Card(null, stripeCard.getExpMonth(), stripeCard.getExpYear(), null, stripeCard.getName(), stripeCard.getAddressLine1(), stripeCard.getAddressLine2(), stripeCard.getAddressCity(), stripeCard.getAddressState(), stripeCard.getAddressZip(), stripeCard.getAddressCountry(), stripeCard.getLast4(), stripeCard.getType(), stripeCard.getFingerprint(), stripeCard.getCountry());
+        return new Card(
+                null,
+                stripeCard.getExpMonth(),
+                stripeCard.getExpYear(),
+                null,
+                stripeCard.getName(),
+                stripeCard.getAddressLine1(),
+                stripeCard.getAddressLine2(),
+                stripeCard.getAddressCity(),
+                stripeCard.getAddressState(),
+                stripeCard.getAddressZip(),
+                stripeCard.getAddressCountry(),
+                stripeCard.getLast4(),
+                stripeCard.getBrand(),
```

Comment:
This line is actually a slight change in the logic -- using getBrand() instead of the deprecated getType() method.